src/components/BulkMetadataForm/__stubs__/schema.ts

```diff
@@ -0,0 +1,34 @@
+import { CustomJsonSchema, MetadataProperties as fields } from '../../../types';
+
+export const schema: CustomJsonSchema = {
```

Comment:
I expected this to be re-usable code. I.e. a helper function which assists in generating the stub schema as well as generating the schema from the `MetadataProperty[]` array that the wizard gets as input from the implementation side. On the implementation end only an array of `MetadataProperty[]` has to be provided so that no knowledge is required about the intricacies of the JsonForms JSON schema.